csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -1250,7 +1250,7 @@
               "title": "Score",
               "description": "specifies information about (at least one) score of the vulnerability and for which products the given value applies.",
               "type": "object",
-              "minItems": 2,
+              "minProperties": 2,
```

Comment:
👀

Reply:
Oh wow! Great catch indeed! Thank you so much!

Reply:
It was actually searching for the mistake I (then) corrected in #310... This was more accidentally.